config/sidekiq.yml

```diff
@@ -0,0 +1,70 @@
+:schedule:
```

Comment:
Just curious if this file is supposed to be separate from `config/sidekiq_production.yml`?

https://github.com/coinfi/coinfi/pull/585/files#diff-3924c9053e4f667358528b94fd4df107

Reply:
No, I made this before making the change in the other pull request. Wasn't sure if the other change would work properly or not so I hadn't updated this file yet.